components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -303,7 +305,18 @@ private void sendNotification(String errorResp, String status, String message,
                                   String acUrl, HttpServletRequest req,
                                   HttpServletResponse resp) throws ServletException, IOException {
 
-        String redirectURL = SAMLSSOUtil.getNotificationEndpoint();
+        if (req.getParameter(SAMLSSOConstants.ECP_ENABLED) == "true") {
+            PrintWriter out = resp.getWriter();
+            try {
+                String soapResp = SAMLSOAPUtils.createSOAPMessage(SAMLSSOUtil.decode(errorResp).
+                        replace("<?xml version=\"1.0\" encoding=\"UTF-8\"?>", ""), acUrl);
+                log.info(soapResp);
```

Comment:
Do we need to log.info responses ?. Doesn't this make logs bulky ?

Reply:
changed to log the response in debug mode